### DISCUSSION TITLE
Tiempo de ejecución

### DISCUSSION THREAD

**Discussion by isufan** [IIC2133-PUC/2024-1#171] [Tarea 2] [answered]
Hola! Cuando terminé la parte 1 de la tarea hace algunos días tuve todos los tests correctos sin ningún timeout y ahora probando nuevamente tengo un timeout en el test hard-3. En todo caso creo que mi computador está un poco lento ahora y además en el tablero del ranking aparece que el mismo test corre en menos de 10 segundos (específicamente en 5.4s), entonces quería saber si es necesario disminuir más el tiempo de ejecución para tener el puntaje de ese test.
Gracias de antemano!

**Comment by sebacarrascop** [comment, marked as answer]
hola!, como dices si antes corría bien y en el ranking está bajo los 10 segundos no deberías tener problemas, puedes probar reiniciando tu computador para tener la menor cantidad de procesos en ejecución para poder probar que efectivamente todo está bien, pero los resultados del ranking son una buena aproximación de los resultados en la evaluación.

**Comment by isufan** [comment]
Muchas gracias por la respuesta, pero me di cuenta que no había cambiado de vuelta el makefile para que optimizara al máximo. Lo cambié y ahora funciona todo bien :)

**Reply by sebacarrascop** [reply]
mejor aún!, ánimo con la tarea
